kubernetes/config/kube_config_yaml.h

```diff
@@ -24,13 +24,15 @@ extern "C" {
  *
  * IN:
  * kubeconfig->fileName: kubernetes cluster configuration file name
```

Comment:
Maybe note that either one, but not both should be supplied?

Reply:
I have added a more explicit note there. There was already a hint in the doc for **kubeconfig->buffer**.
`* IN:
 * kubeconfig->fileName: kubernetes cluster configuration file name
 * kubeconfig->buffer: kubernetes cluster configuration data; **this is considered only if kubeconfig->fileName is set to NULL**
 * 
 * Note: One may use either kubeconfig->fileName or kubeconfig->buffer but not both at the same time.`